src/observer.ts

```diff
@@ -81,7 +72,7 @@ function useForceUpdate() {
     return update
 }
 
-function useObserverReaction(baseComponentName: string) {
+export function useObserver<T>(fn: () => T, baseComponentName = "anonymous"): T {
```

Comment:
Extract it to a separate file, please. And some basic tests would be nice as well :)

Reply:
done, the tests are now the same ones than for observer (except for one case I think)